pkg/api/pod/util.go

```diff
@@ -19,11 +19,11 @@ package pod
 import (
 	"strings"
 
+	"k8s.io/api/core/v1"
```

Comment:
i don't think we can do that. the apparmor feature is considered Beta still:
https://kubernetes.io/docs/tutorials/clusters/apparmor/

can be possibly moved under a separate package, somewhere under:
`k8s.io/api`

cc @tallclair 
/sig auth

Reply:
My reasoning was that the seccomp constants are also beta but they live in k8s.io/api/core/v1. Happy to move them somewhere else, but this seemed like the most appropriate home at the moment.

Reply:
ack, something was telling me that we already have <GA constants in v1.

Reply:
The annotations target core/v1 objects (pods), and are already validated in core/v1 validation, so this is probably the right package for them.